test/unit/node/update.test.ts

```diff
@@ -45,9 +45,11 @@ describe("update", () => {
       return response.end()
     }
 
-    // NOTES subpath match /redirect/${number}
-    // if number === 0, resolve it
-    if (request.url.match(/\/redirect\/[0-9]/)) {
+    // Checks if url matches /redirect/${number}
+    // with optional trailing slash
+    let redirectSlashNumberRegExp = new RegExp(/\/redirect\/([0-9]+)(\/)?$/)
```

Comment:
The `()` around the `\/` could be omitted (unless you want to capture the trailing slash).

Reply:
I think my original thinking was putting everything in capture groups. I'll omit it

Reply:
Makes sense!  It occurred to me you could leave this inline with `match` since `new RegExp` will have a tiny amount of extra overhead (this must be the world's smallest nit lol)

Reply:
LOL I'll leave inline too!